src/contextmenu.js

```diff
@@ -58,7 +66,7 @@ function isPR(linkUrl) {
 
 function parseLink(linkUrl, selectionText, pageUrl) {
     return new Promise((resolve, reject) => {
-        const url = new URL(linkUrl);
+        const url = new URL(linkUrl ? linkUrl : pageUrl);
```

Comment:
```suggestion
        const url = new URL(linkUrl);
```

I'd like to tackle this in a separate PR tbh